docs/backend/vocabularies.md

```diff
@@ -11,3 +11,6 @@ myst:
 
 # Vocabularies
 
+```{seealso}
+See the chapter {ref}`training:vocabularies-label` from the Mastering Plone 6 Training
```

Comment:
This link is broken, too.

https://deploy-preview-1290--6-dev-docs-plone-org.netlify.app/backend/vocabularies.html

Is this the link you want?

https://training.plone.org/5/mastering-plone/registry.html#vocabularies

Reply:
Same

Reply:
Still broken. See screenshot.

![Screen Shot 2022-09-07 at 5 26 05 AM](https://user-images.githubusercontent.com/102112/188877847-82797d2a-ebdc-43f6-bc5a-27d079bf37dc.png)

Reply:
Rebuild fixed the link. Resolving.